release_docs/RELEASE.txt

```diff
@@ -707,6 +755,44 @@ Bug Fixes since HDF5-1.12.0 release
 
         (ADB - 2021/01/19, HDFFV-11106)
 
+    - The tools library was updated by standardizing the error stack process.
+
+        General sequence is:
+          h5tools_setprogname(PROGRAMNAME);
+          h5tools_setstatus(EXIT_SUCCESS);
+          h5tools_init();
+          ... process the command-line (check for error-stack enable) ...
+          h5tools_error_report();
+          ... (do work) ...
+          h5diff_exit(ret);
+
+        (ADB - 2020/07/20, HDFFV-11066)
+
+    - h5diff fixed a command line parsing error.
+
+        h5diff would ignore the argument to -d (delta) if it is smaller than DBL_EPSILON.
+        The macro H5_DBL_ABS_EQUAL was removed and a direct value comparision was used.
+
+        (ADB - 2020/07/20, HDFFV-10897)
+
+    - h5diff added a command line option to ignore attributes.
+
+        h5diff would ignore all objects with a supplied path if the exclude-path argument is used.
+        Adding the exclude-attribute argument will only exclude attributes, with the supplied path,
+        from comparison.
+
+        (ADB - 2020/07/20, HDFFV-5935)
+
+    - h5diff added another level to the verbose argument to print filenames.
+
+        Added verbose level 3 that is level 2 plus the filenames. The levels are:
+          0 : Identical to '-v' or '--verbose'
+          1 : All level 0 information plus one-line attribute status summary
+          2 : All level 1 information plus extended attribute status report
+          3 : All level 2 information plus file names
+
+        (ADB - 2020/07/20, HDFFV-10005)
```

Comment:
From Jira:  HDFFV-10005  Add H5Avlen_get_buf_size AP
Is this a different issue?